tests/integration/data/cassandra/BackendTests.cpp

```diff
@@ -1294,8 +1299,65 @@ TEST_F(BackendCassandraTest, CacheIntegration)
     ASSERT_EQ(done, true);
 }
 
+class MockCacheClassTest : public BackendCassandraTest {
```

Comment:
I highly suggest making current `BackendCassandraTest` a `BackendCassandraTest` class where you don't have `backend_` at all, and then making two simple classes `BackendCassandraTest` and `CacheBackendCassandraTest` that derive from it and define `backend_` field.
This way there will be no overwrite of `backend_` at all.

Reply:
I don't like `MockCacheClassTest` because it's too general, btw, that's why I proposed `CacheBackendCassandraTest` here.